Sources/SwiftfulFirebaseAuth/Providers/FirebaseAuthProvider.swift

```diff
@@ -6,12 +6,23 @@
 //
 
 import Foundation
+import Firebase
 import FirebaseAuth
 
 struct FirebaseAuthProvider: AuthProvider {
     
+    private var auth: Auth
+    
+    init() {
+        self.auth = Auth.auth()
+    }
+    
+    init(firebaseAuth: Auth) {
+        self.auth = firebaseAuth
+    }
```

Comment:
I haven't done this in a project myself. How do you get an instance of Auth other than calling Auth.auth() ? Do you have a sample implementation on how you'd initialize this?